library/core/src/ptr/const_ptr.rs

```diff
@@ -807,7 +808,11 @@ impl<T: ?Sized> *const T {
         }
     }
 
-    /// Adds an offset to a pointer (convenience for `.offset(count as isize)`).
+    /// Adds an offset to a pointer.
```

Comment:
Make this "unsigned offset" or "non-negative offset" for clearer symmetry with the signed APIs?